lib/tasks/duplicate_labware_data_fix.rake

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+namespace :duplicates do
+  desc "De-duplicate labware"
+  task delete: :environment do |_t|
+    ActiveRecord::Base.transaction do
+      puts "Starting de-duping..."
+
+      # Get all the barcodes where there are duplicates
+      barcode_to_count = Labware.group(:barcode).count
+
+      barcode_to_count_duped = barcode_to_count.select { |_barcode, count| count > 1 } # 7106
```

Comment:
Not that it matters too much but you can do
`Labware.group(:barcode).having("count(id) > 1")`